showkase-processor/src/main/java/com/airbnb/android/showkase/processor/models/ShowkaseMetadata.kt

```diff
@@ -52,6 +52,8 @@ internal sealed class ShowkaseMetadata {
         override val enclosingClassName: ClassName? = null,
         override val insideWrapperClass: Boolean = false,
         override val insideObject: Boolean = false,
+        val componentIndex: Int? = null,
+        val componentDistinctName: String? = null,
```

Comment:
since this can be inferred (`"${element.name}_$index"`), maybe we don't need to necessarily save this 🤔

Reply:
Yeah that is a good point! I can just use the `ComponentIndex` instead 😄 I'll fix this, thanks!

Reply:
This should be fixed in [f185cec](https://github.com/airbnb/Showkase/pull/259/commits/f185cec407e6bc8103e3a378726d5b0eb01f215c) 😃 Thanks!